presto-docs/src/main/sphinx/connector/mysql.rst

```diff
@@ -221,10 +221,20 @@ MySQL Connector Limitations
 The following SQL statements are not yet supported:
```

Comment:
```suggestion
The following SQL statements are not supported:
```
In general, technical documentation should use the present tense, and describe only the current state of the software. For references, see the Google developer documentation style guide about [present tense](https://developers.google.com/style/tense), [Document future features](https://developers.google.com/style/future), and [Timeless documentation](https://developers.google.com/style/timeless-documentation).